proto/tendermint/abci/types.proto

```diff
@@ -419,33 +419,31 @@ service ABCIApplication {
 // Side channel related entities
 //
 
-
 //
 // Side channel requests
 //
 
 message RequestBeginSideBlock {
-  bytes hash = 1;
-  tendermint.types.Header header = 2 [(gogoproto.nullable) = false];  
-  repeated tendermint.types.SideTxResult side_tx_results = 3 [(gogoproto.nullable)=false];
+  bytes                   hash                              = 1;
+  tendermint.types.Header header                            = 2 [(gogoproto.nullable) = false];
+  repeated tendermint.types.SideTxResponses side_tx_results = 3 [(gogoproto.nullable) = false];
```

Comment:
Add gogoproto annotations for multi-word keys, ref: https://github.com/cosmos/cosmos-sdk/blob/v0.40.1/proto/cosmos/auth/v1beta1/auth.proto#L45